Aptoide/appcoins-wallet-android
# [APPC-1114] Show no notification in case there's no campaign

**What does this PR do?**

   On a previous ticket, it was added the verification before starting to send proofs, it was decided to show a notification at the time, but it was now discussed that it should not be show any notification like it would not show if the app communicated trough the wallet channel instead of requesting the campaigns. It also fixed the constant sending of the notification every time the user would enter the app.


**Where should the reviewer start?**

WalletPoaService.java

**How should this be manually tested?**

 1. Have a campaign, go trough the end and check if you can convert a campaign.
 2. Don’t have a campaign or already have been rewarded, and check if there’s no notification related to the PoA

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1114

**Code Review Checklist**
